tasks/task07/unittests.py

```diff
@@ -0,0 +1,42 @@
+#tests for Task 07: OrderRepository
+import order_repository, unittest
```

Comment:
1. it is better to import specific classes that you use:
2. it is better to place imports from different libraries on different lines

so that:
```python
import unittest # standard library imports go first

from order_reposiotry import Good, Order, OrderRepository
```